databallpy/load_data/event_data/opta.py

```diff
@@ -397,14 +474,134 @@ def _load_event_data(f24_loc: str, country: str) -> pd.DataFrame:
             pd.to_datetime(event.attrs["timestamp"], utc=True)
         )
 
+        # get extra information for databallpy events
+        if event_name in [
+            "miss",
+            "post",
+            "attempt saved",
+            "not past goal line",
+            "goal",
+            "own goal",
+        ]:
+            shot_events[event.attrs["id"]] = _make_shot_event_instance(
```

Comment:
Should the key be a string or an int? I guess int would be more logical right? Right now it is a string?